dashboard/src/modules/components/ComparisonComponent/PanelContent.jsx

```diff
@@ -67,6 +72,17 @@ const PanelConent = () => {
           )}
         </div>
       )}
+      {/* <RenderPagination
+        items={filteredDatasets.length}
+        page={page}
+        setPage={setPage}
+        perPage={perPage}
+        setPerPage={setPerPage}
+        onSetPage={onSetPage}
+        perPageOptions={perPageOptions}
+        onPerPageSelect={onPerPageSelect}
+      /> */}
```

Comment:
Isn't this a blob of commented-out code?  Do we really want to  commit that??

Reply:
Removed it!